package.json

```diff
@@ -8,7 +8,8 @@
     "build": "npm run clean && tsc",
     "lint": "eslint src/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "clean": "rimraf ./dist"
+    "clean": "rimraf ./dist",
+    "test": "jest --config ./jest.config.json"
```

Comment:
Please  add scripts for coverage and for watch mode

Reply:
Added